bot.py

```diff
@@ -58,14 +58,14 @@
 logger = logging.getLogger('polybot.' + __name__)
 
 
-def main():
-
+def main(args: list[str] = None):
```

Comment:
`list[str]` is an invalid type hint... you've got to use `List` from `typing`

Reply:
This is a feature of Python 3.9, see [PEP 585](https://www.python.org/dev/peps/pep-0585/). If compatibility with 3.7 and 3.8 is required, `from __future__ import annotations` can be used.